src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        final String operationType1 = "supply";
+        final String operationType2 = "buy";
+        final String operationType3 = "result";
+        File importedFile = new File(fromFileName);
+        File exportedFile = new File(toFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(importedFile))) {
+            String data = bufferedReader.readLine();
```

Comment:
The creation of the `data` array is unnecessary. You can directly process `filteredData` without creating a new array, as it already contains the necessary information split by lines.